### PR TITLE
this got so sexy

### PR DESCRIPTION
**Pull request by Apfel**
can this get more sexy

### CODE REVIEW

File: src/main/java/me/artuto/endless/commands/fun/TagCmd.java [hunk 1]
```diff
@@ -270,27 +270,27 @@ protected void executeCommand(CommandEvent event)
                         bot.tdm.deleteGlobalTag(event.getArgs().trim().toLowerCase());
                     else
                         bot.tdm.deleteLocalTag(event.getGuild().getIdLong(), event.getArgs().trim().toLowerCase());
-                    event.replySuccess("Tag successfully deleted");
+                    event.replySuccess("Tag `"+tag.GetName()+"` successfully deleted");
                 }
                 else
```
Comment (Artuto) [inline src/main/java/me/artuto/endless/commands/fun/TagCmd.java]: This is not C#

File: src/main/java/me/artuto/endless/commands/fun/TagCmd.java [hunk 2]
```diff
@@ -270,27 +270,27 @@ protected void executeCommand(CommandEvent event)
...
-                    event.replyError("You aren't the owner of the tag!");
+                    event.replyError("You aren't the owner of the tag `"+tag.GetName()+"`!");
             }
             else
             {
                 Tag tag = bot.endless.getGlobalTag(event.getArgs().trim().toLowerCase());
                 if(tag==null)
                 {
```
Comment (Artuto) [inline src/main/java/me/artuto/endless/commands/fun/TagCmd.java]: Same here

File: src/main/java/me/artuto/endless/commands/fun/TagCmd.java [hunk 3]
```diff
@@ -270,27 +270,27 @@ protected void executeCommand(CommandEvent event)
...
-                    event.replyError("No tag found with that name!");
+                    event.replyError("No tag found with the name `"+tag.GetName()+"`!");
                     return;
                 }
 
                 if(tag.getOwnerId()==event.getAuthor().getIdLong() || event.isOwner())
                 {
                     bot.tdm.deleteGlobalTag(event.getArgs().trim().toLowerCase());
```
Comment (Artuto) [inline src/main/java/me/artuto/endless/commands/fun/TagCmd.java]: And here

File: src/main/java/me/artuto/endless/commands/fun/TagCmd.java [hunk 4]
```diff
@@ -270,27 +270,27 @@ protected void executeCommand(CommandEvent event)
...
-                    event.replySuccess("Tag successfully deleted");
+                    event.replySuccess("Tag "+tag.GetName()+"` successfully deleted");
                 }
                 else
```
Comment (Artuto) [inline src/main/java/me/artuto/endless/commands/fun/TagCmd.java]: Here too

File: src/main/java/me/artuto/endless/commands/fun/TagCmd.java [hunk 5]
```diff
@@ -340,7 +340,7 @@ else if(!(msg.getAttachments().isEmpty()))
                     tag = bot.endless.getGlobalTag(name);
                     if(tag==null)
```
Comment (Artuto) [inline src/main/java/me/artuto/endless/commands/fun/TagCmd.java]: Well you get me

File: src/main/java/me/artuto/endless/commands/fun/TagCmd.java (hunk 1 above)
Comment (Apfel) [reply src/main/java/me/artuto/endless/commands/fun/TagCmd.java]: well C# syntax is this:
```csharp
string woag = "i can be used in another string";
string test = $"cool {woag}";
```

**Comment by Apfel** [comment]
thats how IntelliJ teached me java

**Comment by Artuto** [comment]
I really doubt it

**Comment by Apfel** [comment]
no u

**Comment by Apfel** [comment]
merge this and i get 10% more happy irl

**Comment by Artuto** [comment]
Fix it then

**Comment by Apfel** [comment]
wot

**Comment by Apfel** [comment]
fixed some stuff

**Comment by Apfel** [comment]
build success woag

**Comment by Apfel** [comment]
nice

**Pull request merged**